modules/search-service-impl/src/main/java/org/opencastproject/search/impl/persistence/SearchServiceDatabase.java

```diff
@@ -79,18 +82,17 @@ public interface SearchServiceDatabase {
    * @throws SearchServiceDatabaseException
    *           if there is a problem communicating with the underlying data store
    */
-  MediaPackage getMediaPackage(String mediaPackageId) throws NotFoundException, SearchServiceDatabaseException;
+  MediaPackage getMediaPackage(String mediaPackageId)
+          throws NotFoundException, SearchServiceDatabaseException, UnauthorizedException;
 
   /**
    * Gets media packages from a specific series
    *
-   * @param seriesId
-   *          the series identifier
+   * @param seriesId the series identifier
    * @return collection of media packages
-   * @throws SearchServiceDatabaseException
-   *           if there is a problem communicating with the underlying data store
+   * @throws SearchServiceDatabaseException if there is a problem communicating with the underlying data store
    */
-  Collection<MediaPackage> getMediaPackages(String seriesId) throws SearchServiceDatabaseException;
+  Collection<Pair<Organization, MediaPackage>> getSeries(String seriesId) throws SearchServiceDatabaseException;
```

Comment:
I *strongly* dislike this, but `SearchUpdatedEventHandler` needs to know from which `Organization` a series comes from. Figuring that out some other way (`SeriesService`? Nothing obvious, to me at least) would be better. This can be refactored after merging if needed since *nothing else* needs to know.